### PR TITLE
set-default-values-when-sending-signals-with-variables

### PR DESCRIPTION
**Pull request by oanaBiancaSchiopu**

### CODE REVIEW

**Comment by fviale** [comment]
Retest this please

File: scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/DefaultModelJobValidatorServiceProvider.java [hunk 1]
```diff
@@ -44,6 +44,7 @@
 import org.ow2.proactive.scheduler.common.task.Task;
 import org.ow2.proactive.scheduler.common.task.TaskVariable;
 import org.ow2.proactive.scheduler.common.util.VariableSubstitutor;
+import org.springframework.util.CollectionUtils;
```
Comment (fviale) [inline scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/DefaultModelJobValidatorServiceProvider.java:47]: is it possible to use something else than springframework api here ?

if it only does:

`(variableList != null && !variableList.isempty())`

I would prefer not using it

**Review by fviale — APPROVED** [review]
small remark, otherwise LGTM

**Comment by fviale** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Review by tao-xinxiu — APPROVED** [review]

**Pull request merged**